cluster/gce/gci/configure-helper.sh

```diff
@@ -425,16 +425,94 @@ EOF
   fi
 }
 
+# Write the config for the audit policy.
 function create-master-audit-policy {
-  # This is the config for the audit policy.
-  # TODO(timstclair): Provide a more thorough policy.
-  cat <<EOF >/etc/audit_policy.config
+  local -r path=$1
+
+  # Known api groups
+  local -r known_apis='
+      - group: "" # core
+      - group: "admissionregistration.k8s.io/v1alpha1"
+      - group: "apps/v1beta1"
+      - group: "authentication.k8s.io"
+      - group: "authorization.k8s.io"
+      - group: "autoscaling"
+      - group: "batch"
+      - group: "certificates.k8s.io/v1beta1"
+      - group: "extensions/v1beta1"
+      - group: "networking.k8s.io/v1"
+      - group: "policy/v1beta1"
+      - group: "rbac.authorization.k8s.io"
+      - group: "settings.k8s.io/v1alpha1"
+      - group: "storage.k8s.io"'
+
+  cat <<EOF >$path
 rules:
+  # The following requests were manually identified as high-volume and low-risk,
+  # so drop them.
+  - level: None
+    users: ["system:kube-proxy"]
+    verbs: ["watch"]
+    resources:
+      - group: "" # core
+        resources: ["endpoints", "services"]
+  - level: None
+    users: ["system:unsecured"]
```

Comment:
BTW - This specific request is very high volume. I left an empty cluster sitting over the weekend, and this was the 5th highest request category:

```
$ sqlite3 -header ~/logs/audit-policy/super-audit-2.db "SELECT SUM(count) AS count, user, verb, apigroup, namespace, resource, name, subresource FROM audit WHERE response != '403' GROUP BY user, verb, apigroup, resource, name, subresource ORDER BY count DESC LIMIT 5;" | column -tn -s "|"
count  user                                                     verb    apigroup                   namespace                     resource                name                                subresource
89069  system:kube-controller-manager                           update  core                       kube-system                   endpoints               kube-controller-manager             
89067  system:kube-controller-manager                           get     core                       kube-system                   endpoints               kube-controller-manager             
89063  system:kube-scheduler                                    update  core                       kube-system                   endpoints               kube-scheduler                      
89061  system:kube-scheduler                                    get     core                       kube-system                   endpoints               kube-scheduler                      
71446  system:unsecured                                         get     core                       kube-system                   configmaps              ingress-uid                         
```
(Using https://github.com/timstclair/kube-contrib/blob/devel/devel/scripts/auditdb.go for the database)